Library/Homebrew/download_strategy.rb

```diff
@@ -262,6 +262,10 @@ def initialize(name, resource)
   end
 
   def fetch
+    if @url.include?("https://downloads.sourceforge.net") || @url.include?("https://download.gnome") || @url.include?("https://www.apache.org/dyn/")
```

Comment:
No strong preference from here either. One interesting thing is that `sourceforge.jp` uses a different setup than the main sourceforge site, so that one is probably best to be handled separately.